apps/decoder_gtfs/lib/decoder/gtfs.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Decoder.Gtfs do
+  use Definition, schema: Decoder.Gtfs.V1
+
+  @derive Jason.Encoder
+  defstruct chunk_size: 100
+
+  defimpl Decoder do
+    def lines_or_bytes(_t), do: 2048
+
+    def decode(t, stream) do
+      stream
+      |> Enum.to_list()
+      |> List.flatten()
+      |> Enum.join()
+      |> TransitRealtime.FeedMessage.decode()
```

Comment:
Is this the message we always expect for gtfs? I  cant say I ever fully understood what we did with that since we only ever really had cota.

Reply:
I am not a gtfs expert, but I believe that this is always the message.